airbyte-integrations/connectors/source-sentry/source_sentry/streams.py

```diff
@@ -69,14 +71,21 @@ def __init__(self, *args, **kwargs):
         super(SentryIncremental, self).__init__(*args, **kwargs)
         self._cursor_value = None
 
+    def get_state_value(self, stream_state: Mapping[str, Any] = None) -> str:
+        bad_state_values = [None, "None", ""]
+        state = self.start_date
+        if stream_state:
+            state_value = stream_state.get(self.cursor_field)
+            state = state if state_value in bad_state_values else state_value
+        return pendulum.parse(state)
```

Comment:
Better not to hardcode bad values. 


```suggestion
    def validate_state_value(self, state_value) -> str:   
        none_as_string_or_empty = state_value == "None" if state_value else True
        return self.start_date if none_as_string_or_empty else state_value


    def get_state_value(self, stream_state: Mapping[str, Any] = None) -> str:
        state_value = self.validate_state_value(stream_state.get(self.cursor_field, self.start_date) if stream_state else self.start_date)
        return pendulum.parse(state_value)
```

Reply:
Love it.